lib/dashx/client.rb

```diff
@@ -144,6 +163,10 @@ def save_contacts(uid, contacts = [])
       make_graphql_request(SAVE_CONTACTS_REQUEST, { uid: uid, contacts: contacts })
     end
 
+    def fetch_contacts(uid)
+      make_graphql_request(FETCH_CONTACTS_REQUEST, { accountUid: uid })
```

Comment:
Looking at graphql the parameter that `fetch_contacts` accepts is `uid`. So this should become:
```suggestion
      make_graphql_request(FETCH_CONTACTS_REQUEST, { uid: uid })
```